pilot/pkg/config/clusterregistry/conversion.go

```diff
@@ -67,11 +67,6 @@ func NewClustersStore() *ClusterStore {
 	}
 }
 
-// GetClientAccessConfigs returns map of collected client configs
-//func (cs *ClusterStore) GetClientAccessConfigs() map[string]clientcmdapi.Config {
-//	return cs.clientConfigs
-//}
-
```

Comment:
why did you remove this?

Reply:
we commented out - removing all sorts of unused stuff 💃